### DISCUSSION TITLE
각자 소개 해요!

### DISCUSSION THREAD

**Discussion by bryantson** [Korean-American-IT-Association-KAITA/kaita-project-management#17] [2. 자유토크]
이름: Bryant Son
장소: Dallas, Texas USA
설명: 현재 깃허브에서 Solution Architect 로 일하고 있고 미국에 살고 있습니다. 관심 분야는 KAITA 분들 도와주는 것이고 전문 분야는 깃허브, 데브옵스, Project Management, 클라우드 Architecture 입니다

이름 : 정동근 (Dennis)
지역 : Novi, Michigan US
설명 : Xfinity 안드로이드 앱 개발자
역할 : 안드로이드 멘토링
카톡 ID : topsan

**Comment by devminjae** [comment]
이름: Minjae Yoo
장소: Austin, Texas, US > Dallas로 내년에 이사 계획 중입니다... 

설명: Bioinformatics를 전공했고(BS, MS), Research field에 있다가 industry로 나왔습니다. 
Full-stack > python engineer > DevOps engineer > software engineer 로 일하고 있습니다.

다음 Stack의 목표가 Solution Architect or Data DevOps Engineer 입니다! 무엇이든 재밌는 일이면 환영입니다!!

**Comment by jparksecurity** [comment]
이름: John Park
장소: Dallas, Texas USA
설명: Iowa State University에서 Computer Science 학사 졸업하고 Trelly Group에서 Software Engineer로 일하고 있습니다. 다 같이 즐겁게 배워가면 좋겠습니다.

**Comment by bbkaeul** [comment]
이름: Kyungeun Min
장소: Seoul, Korea
설명: 학부는 경영학과 전공했고, 마케팅에서 일하다 AI engineer가 되기 위해 현재는 AI 석사 공부를 하고 있습니다.  
배우고 있는 언어: 파이썬
많이 부족하지만 열심히 참여해서 많은 실력향상을 하고 싶습니다! 감사합니다.

**Comment by klee5760** [comment]
이름: Kevin Lee(Junghoon Lee)
지역: Champaign, IL, USA
설명: 학부때는 화공학과 전공했고, 요즘 엔지니어도 코딩 경력이 필요해서 코딩 training 받고 있습니다.  
배우고 있는 언어: Java

**Comment by yenny1124** [comment]
이름: Yeeun Choi
장소: Seattle, WA
설명: CS를 전공하고 있는 3학년 대학생입니다. 아직 많이 부족하지만 백엔드 엔지니어를 목표로 하고 있습니다. 이번 프로젝트를 통해 학교 공부와 관련된 것 뿐만 아니라 학교 공부 이외의 것들을 많이 배워가고 싶습니다.
배우고 있는 언어: Java, C++

**Comment by Eunyoungkim0** [comment]
이름: Eunyoung Kim
장소: Charlotte, North Carolina
설명: 한국에서 SI업체에서 2년 일하다 미국에서 다시 시작하고 싶어서 커뮤니티 컬리지부터 다니는 중입니다. (전공 Computer Science 예정) 미국에서 백엔드 developer가 목표이고, 아직 아는게 없는거 같아서.. 이번 프로젝트를 통해 많이 배우고 싶습니다. 잘 부탁드립니다.

**Comment by j2jun** [comment]
이름 : Joseph Jun
지역 : Orange County, CA
설명 : 대학교 2학년이고 CS 전공하고 있습니다. 좋은 추억 만들고 프로젝트로 통해 더 많이 배우고 싶습니다. 잘부탁드립니다!

**Comment by Indosaram** [comment]
서울에서 파이썬 개발하고 있는 윤인도입니다.

**Comment by ghkfehd0502** [comment]
이름: 고태환
장소: Northbrook, IL
설명: HTML, CSS 배우고 파이썬 공부하고 있습니다. 잘 부탁드립니다.

**Comment by ndk6879** [comment]
이름: Donggyun Nam
지역: State College, PA
설명: 대학교 3학년이고 CS전공하고 있습니다. 다른 분들과 프로젝트를 통해 배우고 협업하는 경험을 하고 싶습니다. 잘부탁드립니다 :)  
배우고 있는 언어: Python, C

**Comment by judyyhham** [comment]
이름: Yeonghyeon (Judy) Ham
장소: Toronto, Ontario, Canada
설명: 현재 Computer Science (BSc) 4학년 재학중입니다. 실력을 키우고, 많은 것들을 배워보고 싶습니다. 프로젝트를 시작하려 해도 어디서부터 어떻게 시작해야 할지 몰라 막막 했었는데, 이번 경험이 제게 많은 도움이 될 것 같습니다. 데이터베이스, 백엔드, 프론트엔드, ML, 등등 기회만 된다면 여러 것들을 경험해 제가 무엇을 좋아하고 즐겨하는지 찾고 싶습니다. 열심히 배우겠습니다!

**Comment by haanmo** [comment]
이름: Haan Mo Jong (정한모)
지역: Fort Lauderdale, Florida, USA
설명: The University of Texas at Dallas에서 Software Engineering 박사 받았습니다. 구체적으로 (Non-Functional) Requirements Engineering & High-level Software Architecture 관점에서 blockchain, microservices, cloud를 기업에 합리적으로 적용하는 방법을 전공했습니다. 개발 경험을 더 쌓고 싶어서, 현재는 Citrix에서 Senior Software Engineer로 일하고 있습니다. 클라우드 관련 기술 스택을 쌓아가는 것에 관심이 많습니다.

**Comment by jung-akim** [comment]
이름: Jung-a Kim (김정아)
장소: Plano, TX
설명: San Jose State University에서 통계석사 졸업하구 현재 보험회사에서 데이터사이언티스트로 일하는 중입니다.
배우고 있는 언어: 파이썬, pytorch
하는일: 데이터 분석 - EDA, modeling

**Comment by ghost** [comment]
이름: Jungwon Oh
장소: Orange County, CA
설명: 현재 front-end dev로 일하고있고 PM쪽으로 관심있어서 참여하게되었습니다.

**Comment by yeinsong** [comment]
이름 : Yein Song
지역 : Seoul, Korea
설명 : 게임회사에서 DA로 근무하고 있습니다. 이전에 사업PM으로 근무 이력이 있습니다 :)
회사 업무 외에 관련 분야에서 여러 경험을 쌓고 싶어 참여하게 되었습니다!

**Comment by lhm0421** [comment]
이름: Hyunmyung(Lucas) Lim
장소: Seoul, Korea
설명: 미국에서 전자 학부 및 취업으로 오래 거주했고 유럽에서 오디오/음향 분야로 석사 공부를 하고 올해 한국으로 귀국했습니다. 
배우고 있는 언어:  Python(Librosa, Pytorch) 
하는일: 제조회사에서 오디오 관련 분야로 일을 하고 있습니다. 개인적으로 AI를 활용한 음성/오디오 분야를 공부하고 있습니다.
